### PR TITLE
map internal to external ids in coverage report, fix #212

### LINKED ISSUE #212: Convert sample/group ids in report FIXED
Through query args in chanjo report v2.3.0
`alt_[sample_id]=[display_name]`

### PR DESCRIPTION
**Pull request by robinandeer**
map external and internal ids for the report

### CODE REVIEW

File: scout/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-import logging
-logger = logging.getLogger(__name__)
+from pkg_resources import get_distribution
 
-__version__ = '1.1.3'
+__version__ = get_distribution(__package__).version
```
Comment (moonso) [inline scout/__init__.py]: Finally :smile:

**Comment by moonso** [comment]
Did you rebase this?

**Comment by robinandeer** [comment]
not sure, will check

**Comment by moonso** [comment]
:+1:

**Pull request merged**